File: primerTrimestre/java basico/2alternativas/Ej4alt.java
/**Crea un programa que pida al usuario dos números y muestre su división si el segundo no es cero, 
 * o un mensaje de aviso en caso contrario.
 * 
 * @author Antonio Quesada
 */


import java.util.Scanner;

public class Ej4alt {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos las variables.
    double num1;
    double num2;
    
  //Petición de datos (número 1 y 2).
    System.out.println("Introduce un numero: ");
    num1 = s.nextDouble();    
    System.out.println("Introduce otro numero: ");
    num2 = s.nextDouble();
    
    
    //Proceso.
    if (num2==0) {   //Si número 1 es igual a 0, muestra que no se puede dividir..
      System.out.println("No se puede dividir entre 0. Es un valor indeterminado.");
    } else {   //Si no es así divide ambos números.
        System.out.println("La division es "+(double)(num1/num2));  
    
    }

  }

}

File: primerTrimestre/introduccion_programacion_python/2alternativas_py/ej4alt.py
# Programa: ej4alt.py
# Propósito: Crea un programa que pida al usuario dos números y muestre su división si el segundo no es cero, 
# o un mensaje de aviso en caso contrario.
# Autor: Antonio Quesada
# Fecha: 23/10/2019.
#
# Variables a usar:
#   * num1
#   * num2
#
# Algoritmo:
#   Leemos y pedimos el numero1 y numero2.
#   Si num1 es igual a 0 decimos que el valor de la división es indefinido.
#   Si no escribimos el resultado de la división.

# Petición de datos
num1 = float(input("Introduce un número: "))
num2 = float(input("Introduce otro número: "))
print("------------------------------------------")


# Salida
if num2 == 0:
    print("Valor indefinido. No puedes dividir entre 0")
else:
    print("La división de", num1,"y", num2, "es", num1/num2)
    
